nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -658,6 +667,38 @@ void CollisionMonitor::publishPolygons() const
   }
 }
 
+void CollisionMonitor::toggleCMServiceCallback(
+  const std::shared_ptr<rmw_request_id_t> /*request_header*/,
+  const std::shared_ptr<nav2_msgs::srv::Toggle::Request> request,
+  std::shared_ptr<nav2_msgs::srv::Toggle::Response> response)
+{
+  if (robot_action_prev_.action_type == ActionType::STOP || robot_action_prev_.action_type == ActionType::SLOWDOWN) {
```

Comment:
Why not? I think these are good things to able to toggle if we're in a deadlock condition